bundles/binding/org.openhab.binding.enphaseenergy/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,40 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.enphaseenergy.internal
+Ignore-Package: org.openhab.binding.enphaseenergy.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Enphase Energy Binding
+Bundle-SymbolicName: org.openhab.binding.enphaseenergy
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.7.0.qualifier
+Bundle-Activator: org.openhab.binding.enphaseenergy.internal.EnphaseenergyActivator
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Enphase Energy binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: com.google.common.base,
+ com.google.common.io,
+ org.apache.commons.httpclient,
+ org.apache.commons.httpclient.util,
+ org.apache.commons.lang,
+ org.apache.commons.lang.builder,
+ org.codehaus.jackson,
+ org.codehaus.jackson.annotate,
+ org.codehaus.jackson.map,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.types,
+ org.openhab.io.net.http,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.enphaseenergy
```

Comment:
remove this Export unless other bundles are going to import classes from this binding